managedtenants/utils/ocm.py

```diff
@@ -23,7 +23,7 @@ def retry_hook(exception):
         raise exception
 
 
-class OcmCli:
+class OcmCli:  # pylint: disable=R0904
```

Comment:
I believe it's safe to disable the `too-many-public-methods` globally https://github.com/mt-sre/managed-tenants-cli/blob/main/.linters/pylint#L83

Reply:
Sure, on it :)